NDTensors/src/blocksparse/linearalgebra.jl

```diff
@@ -296,6 +296,102 @@ function LinearAlgebra.eigen(
   return D, V, Spectrum(d, truncerr)
 end
 
+# QR a block sparse Rank 2 tensor.
+#  This code thanks to Niklas Tausendpfund https://github.com/ntausend/variance_iTensor/blob/main/Hubig_variance_test.ipynb
+#
+function LinearAlgebra.qr(T::BlockSparseTensor{ElT,2}; kwargs...) where {ElT}
```

Comment:
Let's import `qr` and change `LinearAlgebra.qr` to just `qr`, since that's the style we've been trying to use in general. I've been adding comments like:
```julia
# Overload of `LinearAlgebra.qr`
```
above the function signatures to clarify where the function originates.

Reply:
Done